src/migrations/00014-modified-at.sql

```diff
@@ -0,0 +1,84 @@
+-- The commented sections of this file must be manually run as they affect models
+-- that are loaded prior to the migration system being available
+
+-- -- migrations.sbvr
+-- ALTER TABLE "migration"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- -- dev.sbvr
+-- ALTER TABLE "model"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- -- permissions.ts
+-- ALTER TABLE "actor"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "api key"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "api key-has-permission"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "api key-has-role"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "permission"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "role"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "user"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "user-has-role"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "user-has-permission"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- resin.sbvr
```

Comment:
This needs to be checked over before merge to make sure any tables that have been added in the meantime also get migrated

Reply:
Up to date as of v0.50.1